.travis.yml

```diff
@@ -6,11 +6,11 @@ before_install:
   - wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key|sudo apt-key add -
   - sudo add-apt-repository ppa:dns/gnu -y
   - sudo apt-get update
-  - sudo apt-get install libmpfr-dev clang-3.4 llvm-3.4 llvm-3.4-dev dragonegg gcc-4.6 gfortran-4.6 autoconf automake build-essential
+  - sudo apt-get install libmpfr-dev clang-3.4 llvm-3.4 llvm-3.4-dev dragonegg-4.7 gcc-4.7 gfortran-4.7 autoconf automake build-essential
 
 install:
   - ./autogen.sh
-  - ./configure --with-dragonegg=/usr/lib/gcc/x86_64-linux-gnu/4.6/plugin/dragonegg.so CC=gcc-4.6 || cat config.log
+  - ./configure --with-llvm=/usr/lib/llvm-3.4/bin/ --with-dragonegg=/usr/lib/gcc/x86_64-linux-gnu/4.7/plugin/dragonegg.so CC=gcc-4.7 || cat config.log
```

Comment:
Travis brokes at configure since it does not find clang-3.4, maybe change --with-llvm=/usr/lib/llvm-3.4/bin/ by --with-llvm=/usr/local/lib/llvm-3.4/bin/